docs/source/developer_documentation/rocky.md

```diff
@@ -1,3 +1,140 @@
+## Rocky
+
+Rocky is part of the openKAT project, made with Django.
+
+## Stack
+
+Django is the framework for this project.
+To comply to government standards, use [Manon](https://github.com/minvws/nl-rdo-manon) for style and accessibility.
+Yarn is used as package manager and ParcelJS is used as bundler to compile the frontend (CSS and Javascript).
+You can find the Manon repository here: [https://github.com/minvws/nl-rdo-manon](https://github.com/minvws/nl-rdo-manon)
+
+## Running Rocky
+
+### Containerized
+
+To run rocky from the docker container, from the parent directory `nl-kat-coordination`, just run:
+
+```bash
+$ make kat
+```
+
+and continue reading this document at "First run".
+
+### Locally
+
+To run rocky locally, follow these steps.
+
+### Installation
+
+Yarn is used to bundle CSS and Javascript.
+You can build Rocky locally using:
+
+```bash
+$ make build
+```
+
+This will set up Django and compile the frontend.
+
+### Running
+
+You can run Rocky using:
+
+```bash
+$ make run
+```
+
+## First run
+
+After running the first time, visit [localhost:8000](http://localhost:8000) in your browser.
+Log in with credentials: admin / admin
+
+You will be prompted to create secure your account with a One Time Password, so get your authenticator ready.
+
+## Testing
+
+To run all tests, run:
+
+```bash
+$ make test
+```
+
+## Database
+
+To connect to the PostgreSQL database, set the following environment variables (e.g. "localhost", "5432" etc.):
+
+```
+ROCKY_DB_HOST=
+ROCKY_DB_PORT=
+ROCKY_DB=
+ROCKY_DB_USER=
+ROCKY_DB_PASSWORD=
+```
+
+
+## KATalogus View Structure
```

Comment:
Since I moved these diagrams in another PR as well, there is a duplicate for the Katalogus views diagram now (see below)!